README.md

```diff
@@ -153,11 +149,11 @@ $ kubectl create -f deployments/sriov-crd.yaml
 This section explains an example deployment of SR-IOV Network device plugin in Kubernetes if you choose DANM as your meta plugin.
 
 ##### Install DANM
-Refer to [DANM documentation](https://github.com/nokia/danm#getting-started) for detailed instructions.
+Refer to [DANM deployment documentation](https://github.com/nokia/danm#deployment) for detailed instructions.
```

Comment:
nit: you can reference deployment guide directly 

https://github.com/nokia/danm/blob/master/deployment-guide.md